liferay-lima/liferay-portal
# Lps 202517

Issues: 
https://liferay.atlassian.net/browse/LPS-202516 Add the new form to get Images from Dall-e 
https://liferay.atlassian.net/browse/LPS-202517 Create the backend logic to call the client (by now the mock client with testing porpouses) 


### How to test it: 
See https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2413429012/How+to+use+the+OpenAI+mock+client for reference. 
1) Set VALID_API_KEY as Apy key for AI Creator in Instance Settings
2) Enable mock Client via Gogo shell (scr:enable com.liferay.ai.creator.openai.web.internal.client.MockAICreatorOpenAIClient)
3) Go to D&M and click Create AI Image
4) On Description, add USER_IMAGES_URL_ + an url that exists. i.e USER_IMAGES_URL_https://fastly.picsum.photos/id/1/5000/3333.jpg?hmac=Asv2DU3rA_5D1xSe22xZK47WEAN0wjWeFOhzd13ujW4
